pyramid/config/views.py

```diff
@@ -762,6 +776,12 @@ def view(context, request):
         if context is None:
             context = for_
 
+        isexc = isexception(context)
+        if exception_only and not isexc:
+            raise ConfigurationError(
+                'view "context" must be an exception type when '
+                '"exception_only" is True')
```

Comment:
Is there a technical reason that the context for an exception view has to be an exception? Just wondering, not that I see any reason why it shouldn't always be an exception...